### PR TITLE
Wet-boew working examples #4

### PR DESCRIPTION
**Pull request by SebastianBurke**
Added wet-boew working examples::

charts
details-close
dismissable-content
exitscript
tables
wb-disable
wb-postback

### CODE REVIEW

File: wet-boew/charts/index.json-ld [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+	},
+	"description": {
+		"en": "Dynamically generates charts and graphs from table data",
+		"fr": "Génère dynamiquement des graphiques et des diagrammes à partir des données de tableaux"
+	},
+	"modified": "2024-05-16",
+	"componentName": "charts",
+	"status": "stable",
+	"pages": {
+		"examples": [
```
Comment (duboisp) [inline wet-boew/charts/index.json-ld:21]: Please add the other examples
* Customization
* Customizing pie charts
* Labels and reference values
* Specific test cases

File: wet-boew/equalheight/index.json-ld [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+	},
+	"description": {
+		"en": "Equalises the height of elements that are on the same baseline",
+		"fr": "Égalise la hauteur d'éléments étant sur la même ligne de base"
+	},
+	"modified": "2024-05-16",
+	"componentName": "equalheight",
+	"status": "stable",
+	"pages": {
+		"examples": [
```
Comment (duboisp) [inline wet-boew/equalheight/index.json-ld:21]: Missing one example, the one that use CSS grid and put it first in the JSON array.

File: wet-boew/details-close/index.json-ld [hunk 3]
```diff
@@ -0,0 +1,46 @@
...
+		"en": "Detail closed",
+		"fr": "Details fermée"
+	},
+	"description": {
+		"en": "Keep selected details elements closed on defined viewport and down if they were not engaged, default is small",
+		"fr": "Maintient les éléments de détails sélectionnés fermés sur le viewport défini et en dessous s'ils n'ont pas été utilisés, par défaut sur petit"
+	},
+	"modified": "2024-05-16",
+	"componentName": "details-close",
+	"status": "stable",
```
Comment (duboisp) [inline wet-boew/details-close/index.json-ld:19]: Change its status to "provisional"

File: wet-boew/tables/index.json-ld [hunk 4]
```diff
@@ -0,0 +1,46 @@
...
+	},
+	"description": {
+		"en": "Integrates the DataTables plugin into WET providing searching, sorting, filtering, pagination and other advanced features for tables",
+		"fr": "Ajoutes des fonctionnalités aux tableaux telles que la recherche, le tri, le filtrage et la pagination"
+	},
+	"modified": "2024-05-17",
+	"componentName": "tables",
+	"status": "stable",
+	"pages": {
+		"examples": [
```
Comment (duboisp) [inline wet-boew/tables/index.json-ld:21]: Please add the other example: Table filter

**Review by duboisp — DISMISSED** [review]
See the inline comments.

File: wet-boew/charts/index.json-ld [hunk 5]
```diff
@@ -0,0 +1,86 @@
...
+		"docs": [
+			{
+				"title": "Charts and graphs - Simple and easy",
+				"language": "en",
+				"url": "https://wet-boew.github.io/wet-boew/docs/ref/charts/charts-en.html"
+			},
+			{
+				"title": "Graphiques - Simple",
+				"language": "fr",
+				"url": "https://wet-boew.github.io/wet-boew/docs/ref/charts/charts-en.html"
```
Comment (Garneauma) [inline wet-boew/charts/index.json-ld:82]:
```suggestion
				"url": "https://wet-boew.github.io/wet-boew/docs/ref/charts/charts-fr.html"
```

File: wet-boew/details-close/index.json-ld [hunk 6]
```diff
@@ -0,0 +1,46 @@
+{
+	"@context": {
+		"@version": 1.1,
+		"dct": "http://purl.org/dc/terms/",
+		"title": { "@id": "dct:title", "@container": "@language" },
+		"description": { "@id": "dct:description", "@container": "@language" },
+		"modified": "dct:modified"
+	},
+		"title": {
+		"en": "Detail closed",
```
Comment (Garneauma) [inline wet-boew/details-close/index.json-ld:10]:
```suggestion
		"en": "Details closed",
```

File: wet-boew/details-close/index.json-ld [hunk 7]
```diff
@@ -0,0 +1,46 @@
...
+		"en": "Keep selected details elements closed on defined viewport and down if they were not engaged, default is small",
+		"fr": "Maintient les éléments de détails sélectionnés fermés sur le viewport défini et en dessous s'ils n'ont pas été utilisés, par défaut sur petit"
+	},
+	"modified": "2024-05-16",
+	"componentName": "details-close",
+	"status": "provisional",
+	"pages": {
+		"examples": [
+			{
+				"title": "Detail closed",
```
Comment (Garneauma) [inline wet-boew/details-close/index.json-ld:23]:
```suggestion
				"title": "Details closed",
```

File: wet-boew/details-close/index.json-ld [hunk 8]
```diff
@@ -0,0 +1,46 @@
...
+				"language": "en",
+				"url": "https://wet-boew.github.io/gcweb-compiled-demos/wetboew-demos/details-close/details-close-en.html"
+			},
+			{
+				"title": "Details fermée",
```
Comment (Garneauma) [inline wet-boew/details-close/index.json-ld:28]:
```suggestion
				"title": "Details fermé",
```

File: wet-boew/details-close/index.json-ld [hunk 9]
```diff
@@ -0,0 +1,46 @@
...
+	"@context": {
+		"@version": 1.1,
+		"dct": "http://purl.org/dc/terms/",
+		"title": { "@id": "dct:title", "@container": "@language" },
+		"description": { "@id": "dct:description", "@container": "@language" },
+		"modified": "dct:modified"
+	},
+		"title": {
+		"en": "Detail closed",
+		"fr": "Details fermée"
```
Comment (Garneauma) [inline wet-boew/details-close/index.json-ld:11]:
```suggestion
		"fr": "Details fermé"
```

File: wet-boew/exitscript/index.json-ld [hunk 10]
```diff
@@ -0,0 +1,46 @@
...
+		"title": { "@id": "dct:title", "@container": "@language" },
+		"description": { "@id": "dct:description", "@container": "@language" },
+		"modified": "dct:modified"
+	},
+		"title": {
+		"en": "Exit script",
+		"fr": "Sortie de navigation avisé"
+	},
+	"description": {
+		"en": "Redirects users to another sites",
```
Comment (Garneauma) [inline wet-boew/exitscript/index.json-ld:14]:
```suggestion
		"en": "Redirects users to another site",
```

File: wet-boew/tables/index.json-ld [hunk 11]
```diff
@@ -0,0 +1,56 @@
...
+		"description": { "@id": "dct:description", "@container": "@language" },
+		"modified": "dct:modified"
+	},
+		"title": {
+		"en": "Tables",
+		"fr": "Tables"
+	},
+	"description": {
+		"en": "Integrates the DataTables plugin into WET providing searching, sorting, filtering, pagination and other advanced features for tables",
+		"fr": "Ajoutes des fonctionnalités aux tableaux telles que la recherche, le tri, le filtrage et la pagination"
```
Comment (Garneauma) [inline wet-boew/tables/index.json-ld:15]:
```suggestion
		"fr": "Ajoute des fonctionnalités aux tableaux telles que la recherche, le tri, le filtrage et la pagination"
```

**Review by Garneauma — CHANGES_REQUESTED** [review]

**Review by Garneauma — CHANGES_REQUESTED** [review]

**Pull request merged**